### ISSUE TITLE
Create a Docker environment for easier setup

### ISSUE THREAD

**Issue by fabd** [fabd/kanji-koohii#107]
Working on it...

- **Docker** http://www.docker.com

Good tutorial for docker-compose

- https://perchrunway.com/blog/2017-01-19-getting-started-with-docker-for-local-development

PHP images

- https://store.docker.com/images/php

**Comment by fabd** [comment]
Optional:

- [ ]  Use a smaller php/apache image (than the official PHP one)

**Comment by fabd** [comment]
Abandoned.

There is no standard Docker setup, as OS X uses a VM under the hood with no native support for containers (it's unix , not linux based). I tried Docker with a shared folder and the page load takes 500ms minimum on average. File sharing is über slow on OS X.

I tried docker-osx-dev to solve that (which uses rsync), and it's a pain in the ass. There are a bunch of [alternatives](https://github.com/brikis98/docker-osx-dev#alternatives), each of them with their own quirks.

It's just not worth the trouble for me. I'll use MAMP, which is pretty much the same anyway, providing an isolated php/mysql/apache environment. It's like a pre-made container and that's fine it does what we need so that's good for me. I'm not a sever guy , and every hour I spend configuring this junk is infuriating to me when I think I could invest this time on Koohii.

All I wanted was to be able to work in OS X (host) instead of within my Ubuntu virtual machine which I previously used in Windows. I don't want to configure LAMP in my host OS, because I want this environment to be portable. The MAMP package should solve this.
